src/language-js/printer-estree.js

```diff
@@ -218,7 +218,10 @@ function printPathNoParens(path, options, print, args) {
         return [printDirective(node.expression, options), semi];
       }
 
-      if (options.parser === "__vue_event_binding") {
+      if (
+        options.parser === "__vue_event_binding" ||
+        options.parser === "__vue_ts_event_binding"
```

Comment:
What's the benefit of making a difference between `__vue_...` and `__vue_ts_...`?
Is it for performance? If not, I assume the ts version should just be used, because it has alle the features that js covers but more, does it?

Reply:
https://github.com/prettier/prettier/pull/12584#discussion_r851579758